ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/lists/components/content/internal/listItem/clientlibs/styles/listItem.less

```diff
@@ -0,0 +1,24 @@
+.aembox-list {
+    .aem-Grid {
+        display: flex;
+        flex-direction: column;
+
+        .item-component {
+            display: flex;
+            margin: .3rem;
+            background-color: #f5f5f5;
+            font-family: adobe-clean, Helvetica, Arial, sans-serif;
+
+            .item-property {
+                padding: .5rem;
```

Comment:
Please consider putting greate left and right paddings to compensate visual disparity with top/bottom paddings)

Reply:
thanks for the suggestion. these paddings seem to be visually equal (perhaps it is some kind of optical illusion), so I think I'll leave them as is for now :)